### PR TITLE
added 42Istanbul

### PR DESCRIPTION
**Pull request by bertt6**
**Describe the pull request**

**Checklist**

- [ ] I have made the modifications or added tests related to my PR
- [ ] I have run the tests and linters locally and they pass
- [ ] I have added/updated the documentation for my RP

**Additional context**

### CODE REVIEW

**Comment by bertt6** [comment]
Added 42Istanbul Clusters

**Comment by 42Atomys** [comment]
@bertt6 hello, thanks for your contribution, you forgot to run linters on your contribution (logs: https://github.com/42Atomys/stud42/actions/runs/5106707296/jobs/9179074008?pr=452)

Please run it `yarn make-pretty` to auto format your code. 
`yarn style:all` to check of your modifications is aligned with the project :)

File: web/ui/package.json
[the remark this answers is not in the document]
Comment (42Atomys) [reply web/ui/package.json:1]: You have done an update of few packages on `package.json` file, and this is not relative to `add istanbul` scope, 

Due to this update a very huge change is added to the PR Screenshot 2023-05-31 at 12 41 21

This not be inside of this pull request (add istanbul =/= update dependencies). You need to revert this changes to only keep changes relative to `add istanbul` scope

**Comment by bertt6** [comment]
If i don't use yarn install i can't "yarn make-pretty". So I have to write "yarn install". And if i write "yarn install", yarn is adding and changing a lot of file.

**Comment by 42Atomys** [comment]
@bertt6 I encourage you to read and follow the contributing file https://github.com/42Atomys/stud42/blob/main/CONTRIBUTING.md all things to know is listed on it :)

**Comment by bertt6** [comment]
I have already read. So, what should i do right now?

**Comment by 42Atomys** [comment]
I think you don't use the devcontainer feature, you use the project directly on your host

**Comment by 42atomys** [comment]
Due to a long time of inactivity, this pull request has been closed. 
Create a new one to implement your changes again

Best regards!

**Pull request closed without merging**